examples/databricks/log_runs.py

```diff
@@ -0,0 +1,45 @@
+"""
+Logs MLflow runs in Databricks.
+
+How to run:
+$ python examples/databricks/log_runs.py --host <host> --token <token> --user <user>
+
+See also:
+https://docs.databricks.com/dev-tools/api/latest/authentication.html#generate-a-personal-access-token
+"""
+import os
+import uuid
+import argparse
+
+from sklearn.linear_model import LinearRegression
+from sklearn.datasets import load_iris
+
+import mlflow
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--host")
+    parser.add_argument("--token")
+    parser.add_argument("--user")
```

Comment:
Shall we also allow the user to specify an experiment ID?

Reply:
Sure, I'll add it as an optional argument :)